swift/WorkflowTesting/Sources/WorkflowRenderTester.swift

```diff
@@ -151,6 +151,15 @@ fileprivate final class RenderTestContext<T: Workflow>: RenderContextType {
         return workflow.render(state: testContext.state, context: context)
     }
 
+    func makeSink<Action>(of actionType: Action.Type) -> Sink<Action> where Action : WorkflowAction, T == Action.WorkflowType {
+        let (signal, observer) = Signal<AnyWorkflowAction<WorkflowType>, NoError>.pipe()
```

Comment:
For the render tester, it's the same implementation as before `makeSink` was a real function. However, all of this is synchronous in the tester, so it will behave appropriately.

I want to revisit the tester after this change anyway, since this should allow us expect/assert specific concrete Action types (which couldn't be done before, since they were wrapped in the type erasure `AnyWorkflowAction`). But will likely be a followup.